src/django_idom/websocket/consumer.py

```diff
@@ -64,7 +64,7 @@ async def _run_dispatch_loop(self):
             ),
             carrier=WebsocketConnection(self.close, self.disconnect, dotted_path),
         )
-        component_args: tuple[Any, ...] = tuple()
```

Comment:
lowercase tuple is actually fine if you add `from __future__ import annotations`